packages/3id-did-resolver/generateVectorDocuments.js

```diff
@@ -123,8 +123,9 @@ const legacyDid = async (threeId, threeIdGenesisCopy, ceramic) => {
   // uses the first public key of the seed for the v1 threeId as the keydid.
   // sorry for the magic here -.-
   await ceramic.setDIDProvider(threeIdGenesisCopy.getDidProvider())
-  const docParams =  { deterministic: true, metadata: { controllers: [firstKeyDid], family: '3id' } }
-  const doc = await ceramic.createDocument('tile', docParams, { anchor: false, publish: false })
+  const metadata = { controllers: [firstKeyDid], family: '3id', deterministic: true }
+  const doc = await TileDoctype.create(
+      ceramic, {}, metadata, { anchor:false, publish: false })
```

Comment:
Is there a difference here between passing undefined (what happened before) and an empty object?

Reply:
Yeah, this should probably have been `null`. Updated